package.json

```diff
@@ -218,8 +216,10 @@
     "lint:mixedindent": "mixedindentlint --ignore-comments \"client/**/*.scss\" \"assets/**/*.scss\" \"**/*.js\" \"**/*.jsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/config/index.js\"",
     "precommit": "npm run -s install-if-no-packages && node bin/pre-commit-hook.js",
     "prepush": "npm run -s install-if-no-packages && node bin/pre-push-hook.js",
-    "rm": "node bin/rm.js",
-    "prestart": "npm run -s install-if-deps-outdated && node bin/welcome.js && node bin/version-check.js",
+	"rm": "node bin/rm.js",
+	"prestart:check-version": "npx check-node-version --node \"^10.5\" --npm \"~6\"",
+	"prestart:welcome": "node bin/welcome.js",
+    "prestart": "run-s prestart:check-version install-if-deps-outdated prestart:welcome",
```

Comment:
No need to use `run-s`, `cmd.exe` supports the `&&` notation :)

Reply:
in this case, I actually like it. The command with all of the `&&` was getting noisy.

Reply:
Oh it just occured to me, `run-s` isn't available on a clean install (without `node_modules`) 🤦‍♂️